app/views/about/privacy_consent.en.html.erb

```diff
@@ -0,0 +1,78 @@
+<% title 'Privacy consent notice' %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <div class="grid_content_header">
+      <h1 class="app__heading  heading-xlarge gv-u-heading-xxlarge">Privacy notice</h1>
+    </div>
+      <section id="privacy-notice" class="moj-Section moj-Section--2" data-block-name="privacy-notice">
+        <h3 class="heading-small">Being contacted about your experience of using this service</h3>
+
+        <p>By choosing to be contacted by Ministry of Justice (MoJ) you agree that your personal details and any information you give can be used by MoJ for research purposes.</p>
+
+        <h3 class="heading-small">The information we use</h3>
```

Comment:
Let's try to maintain the same styling and markup we have in the other privacy page.
So for each section, it will be something like:

```ruby
    <section class="moj-Section about-section">
      <h2 class="moj-Section__heading moj-Section__heading--2 gv-u-heading-xlarge">section title</h2>

      <div class="Section__content govuk-govspeak gv-s-prose">
        <p>section content</p>
      </div>
    </section>
```